src/common/components/signInForm/signInForm.tpl.html

```diff
@@ -35,18 +35,7 @@
   ng-if="!$ctrl.onSignInPage"
   ng-click="$ctrl.onSignUpWithOkta()"
 >
-  <!-- Circular Okta aura inlined so that the color can be changed with CSS for the :hover and :focus states -->
-  <svg width="24" height="24" viewBox="0 0 16 16" aria-label="Okta logo">
-    <circle
-      cx="8"
-      cy="8"
-      r="5.25"
-      stroke="currentColor"
-      stroke-width="3.5"
-      fill="none"
-    />
-  </svg>
-  <span translate>Sign up with Okta</span>
```

Comment:
Is this a change that Give owners have approved? `Sign up with Okta` gives clarity that the user will be redirected to Okta.